domain/src/test/java/org/oppia/android/domain/question/QuestionAssessmentProgressControllerTest.kt

```diff
@@ -111,6 +111,25 @@ class QuestionAssessmentProgressControllerTest {
     profileId1 = ProfileId.newBuilder().setInternalId(1).build()
   }
 
+  @Test
+  fun testEphemeralState_startExploration_shouldIndicateNoButtonAnimation() {
+    setUpTestApplicationWithSeed(questionSeed = 0)
+    startSuccessfulTrainingSession(TEST_SKILL_ID_LIST_012)
+    val ephemeralQuestion = waitForGetCurrentQuestionSuccessfulLoad()
+    assertThat(ephemeralQuestion.ephemeralState.showContinueButtonAnimation).isEqualTo(false)
+  }
+
+  @Test
+  fun testEphemeralState_moveToNextState_shouldIndicateNoButtonAnimation() {
+    setUpTestApplicationWithSeed(questionSeed = 0)
+    startSuccessfulTrainingSession(TEST_SKILL_ID_LIST_012)
+    waitForGetCurrentQuestionSuccessfulLoad()
+    submitCorrectAnswerForQuestion0()
+    moveToNextQuestion()
+    val ephemeralQuestion = waitForGetCurrentQuestionSuccessfulLoad()
+    assertThat(ephemeralQuestion.ephemeralState.showContinueButtonAnimation).isEqualTo(false)
```

Comment:
```suggestion
    assertThat(ephemeralQuestion.ephemeralState.showContinueButtonAnimation).isFalse()
```

Ditto elsewhere in this test suite & in ExplorationProgressControllerTest (including for true cases).

Reply:
done